src/components/NavBar.vue

```diff
@@ -215,6 +235,19 @@ export default {
       { icon: 'info', text: 'About Ambianic', link: '/about' }
     ]
   }),
+  methods: {
+    setConnectionTooltipText () {
+      if (this.peerConnectionStatus === 'PEER_DISCONNECTED') {
+        this.connectionStatusTooltipText = 'Disconnected'
+        this.connectionStatusIcon = 'cloud-off-outline'
+        this.connectionIconColor = 'warning'
+      } else if (this.peerConnectionStatus === 'PEER_CONNECTING') {
+        this.connectionStatusIcon = 'cloud-sync-outline'
+        this.connectionIconColor = 'black'
```

Comment:
@vickywane let's use default colors such as `warning` and `info` to avoid problems when the OS switches between light and dark themes. We had this issue in the past several times when some contributors pick colors they like for their widgets which do not agree with the colors of widgets elsewhere in the app.